Updating KRX data...
Updating between dates: 2025-05-02 / 2025-05-29
No new data to update
Traceback (most recent call last):
  File "/home/andy/projects/trader/data_collection/dc05_CompanyHealth.py", line 378, in <module>
    update_main_db(log_file, main_db, df_krx, plot_gen_control_file)
  File "/home/andy/projects/trader/data_collection/dc05_CompanyHealth.py", line 306, in update_main_db
    full_rescan_code, partial_rescan_code = _generate_update_codelist(log_file, start_day, end_day)
ValueError: not enough values to unpack (expected 2, got 0)
updating market prices...
Traceback (most recent call last):
  File "/home/andy/projects/trader/data_collection/dc06_GenPriceDB.py", line 172, in <module>
    gen_market_DB(price_DB_path, volume_DB_path, START_DATE, log_file=log_file)
  File "/home/andy/projects/trader/data_collection/dc06_GenPriceDB.py", line 70, in gen_market_DB
    res = fdr.DataReader(code, START_DATE)[['Close', 'Volume']]
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/FinanceDataReader/data.py", line 84, in DataReader
    return NaverDailyReader(codes, start, end).read()
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/FinanceDataReader/naver/data.py", line 33, in read
    return _naver_data_reader(self.code, self.start, self.end)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/FinanceDataReader/naver/data.py", line 9, in _naver_data_reader
    r = requests.get(url + symbol)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/requests/api.py", line 73, in get
    return request("get", url, params=params, **kwargs)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/requests/api.py", line 59, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 467, in _make_request
    self._validate_conn(conn)
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/urllib3/connectionpool.py", line 1096, in _validate_conn
    conn.connect()
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/urllib3/connection.py", line 642, in connect
    sock_and_verified = _ssl_wrap_socket_and_match_hostname(
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/urllib3/connection.py", line 782, in _ssl_wrap_socket_and_match_hostname
    ssl_sock = ssl_wrap_socket(
  File "/home/andy/projects/trader/venv/lib/python3.10/site-packages/urllib3/util/ssl_.py", line 445, in ssl_wrap_socket
    context.load_verify_locations(ca_certs, ca_cert_dir, ca_cert_data)
KeyboardInterrupt

[?2004h(venv) ]0;andy@andy-ThinkPad-T450s: ~/projects/trader/data_collection[01;32mandy@andy-ThinkPad-T450s[00m:[01;34m~/projects/trader/data_collection[00m$ 